crypto/fipsmodule/rand/new_rand_prefix.h

```diff
@@ -0,0 +1,21 @@
+// Prefix some symbols.
+// Delete when promoting "new rand" to default choice.
+
+#ifndef OPENSSL_HEADER_CRYPTO_RAND_LOCAL_PREFIX_INTERNAL_H
+#define OPENSSL_HEADER_CRYPTO_RAND_LOCAL_PREFIX_INTERNAL_H
+
+#include <openssl/base.h>
+
+#define NR_CONCAT(x,y) x##_##y
+#define NR_PREFIX(x) NR_CONCAT(new_rand,x)
+
+#endif // OPENSSL_HEADER_CRYPTO_RAND_LOCAL_PREFIX_INTERNAL_H
```

Comment:
Doesn't this need to be at the end or you could end up with multiple definitions of the below aliases?

Reply:
No, see your second comment for this file.